### DISCUSSION TITLE
Ideas  for 💡 Twitter hashtags to promote and showcase the brilliance of this incredible bootcamp.

### DISCUSSION THREAD

**Discussion by xoraus** [koushikkothagal/java-backend-bootcamp#24] [Ideas]
I'm incredibly grateful to our instructor (@koushikkothagal) for taking the initiative to create this amazing bootcamp and impart his vast knowledge to the next generation of tech professionals.

- #JavaBrainsEmpowerment
- #JavaBrainsUnleashed
- #JobReadyJava
- #TechFutureReady
- #JavaProdigy
- #JavaMastermind
- #JavaJumpstart
- #JavaJediBootcamp
- #DevLoverJavaBrains
- #RecessionProofJava
- #Zero2JobBootcamp

**Comment by koushikkothagal** [comment]
Thanks for the thoughtfulness of this post. I love the idea of hashtags for spreading the word. How about, very simply, #JavaBrainsBootcamp ?
I do like #JobReadyJava as well. Maybe we can vote on one during our next session :)
